ansible/roles/k8s_setup_longhorn/tasks/multipath-disallow.yml

```diff
@@ -0,0 +1,28 @@
+- name: Read the multipath configuration
+  ansible.builtin.command: multipath -t
+  changed_when: false
+  register: cmd_multipath
+
+- name: Parse the configuration directory value
+  ansible.builtin.set_fact:
+    multipath_config_dir: "{{ cmd_multipath['stdout'] | regex_findall('config_dir \"(.+?)\"') | last }}"
+  when: cmd_multipath is not skipped
```

Comment:
Is the multipath config directory dynamic? Why do you search for it in the `multipath -t` command?
As far as I'm concerned this should always be located in `/etc/multipath/conf.d/`

Reply:
It can be configured, with `/etc/multipath/conf.d` being the default value. `man multipathd.conf` says it's a deprecated option, but I'm not sure if that means the path will become non-configurable or the .d directory won't be supported anymore. I opted for adhering to the current mechanism. When the option gets removed, we will notice and fix the error.

As an alternative and probably better approach, I could hardcode /etc/multipath/conf.d/ and then use `multipath -t` to verify that the change has been applied.